### PR TITLE
Use `pointerDown` to know if pointer is down.

### PR DESCRIPTION
**Pull request by cdata**

### CODE REVIEW

File: iron-button-state.html [hunk 1]
```diff
@@ -58,8 +58,18 @@
...
         observer: '_activeChanged'
-      }
+      },
 
+      /**
+       * True if the element is currently being pressed by a "pointer," which
+       * is loosely defined as mouse or touch input (but specifically excluding
+       * keyboard input).
+       */
+      pointerDown: {
```
Comment (notwaldorf) [inline iron-button-state.html]: 👇

File: iron-button-state.html [hunk 2]
```diff
@@ -91,16 +101,20 @@
     },
 
     _downHandler: function() {
+      this._setPointerDown(true);
```
Comment (notwaldorf) [inline iron-button-state.html]: 'down' only means 'mousedown' right? that's the promise here? :)

**Comment by notwaldorf** [comment]
lgtm! i'm not a giant fan of the `pointerDown` name, but I don't have a better one so run with it :)

File: iron-button-state.html (hunk 2 above)
Comment (cdata) [inline iron-button-state.html]: It also means touch down.

**Pull request merged**